rhine-cassava/src/FRP/Rhine/Clock/Csv.hs

```diff
@@ -0,0 +1,45 @@
+module FRP.Rhine.Clock.Csv (
+  CsvClock,
+  csvClock,
+)
+where
+
+-- transformers
+import Control.Monad.Trans.Except
+
+-- bytestring
+import Data.ByteString
+
+-- vector
+import Data.Vector
+
+-- cassava
+import Data.Csv
+
+-- rhine
+import FRP.Rhine (GetClockProxy)
+import FRP.Rhine.Clock
+import FRP.Rhine.Clock.File
+
+{- | Similar to 'File', but ticks at every line of a CSV file,
+and returns a parsed record.
+
+Additionally to an end-of-file error, it can also return a parse error as a 'String'.
+-}
+newtype CsvClock = CsvClock (File String (Vector Record))
```

Comment:
It would be nice if the clock automatically did the parsing (and throws an error if it fails)